__tests__/app.test.js

```diff
@@ -131,3 +131,44 @@ describe("GET /api/articles", () => {
       })
   })
 })
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: responds with an array of comments for the given article_id, sorted by most recent first", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toBeInstanceOf(Array)
+        expect(comments).toHaveLength(11)
+        comments.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: 1,
+            })
+          )
+        })
+        expect(comments).toBeSortedBy("created_at", { descending: true })
+      })
+  })
+  test("404: responds with 'No comments for this article' if article has no comments", () => {
+    return request(app)
```

Comment:
this article doesnt have comments because the article itself doesnt exist - what about when the article exists but has no comments posted to it yet? what should we return in this case? (hint, think about an newly posted article whihc just hasn't been commented on - is that a bad thing?)